### PR TITLE
Release notes for 3.4.6-1.6

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: source/release_notes/3.4.6-1.6.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.. _3.4.6-1.6:
+
+===================
+ |PSMDB| 3.4.6-1.6
+===================
+
+:Date: July 26, 2017
```
Comment (denis-protivenskii) [inline source/release_notes/3.4.6-1.6.rst:7]: Why is there no general PSMDB description as in 3.2.15?

Comment (azhebel) [reply source/release_notes/3.4.6-1.6.rst:7]: Added. The format of the RNs is slightly different.

File: source/release_notes/3.4.6-1.6.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+:Download: http://www.percona.com/downloads/percona-server-mongodb-3.4/
+:Installation: :ref:`install`
+
+
+|PSMDB| is a highly scalable,
+zero-maintenance downtime database
+supporting the MongoDB v3.2 protocol and drivers.
```
Comment (denis-protivenskii) [inline source/release_notes/3.4.6-1.6.rst]: v3.4 ?

Comment (azhebel) [reply source/release_notes/3.4.6-1.6.rst]: Fixed

**Comment by denis-protivenskii** [comment]
LGTM

**Review by igorsol — APPROVED** [review]
LGTM

**Pull request merged**